osu.Game/Screens/Play/HUD/DefaultKeyCounterDisplay.cs

```diff
@@ -42,6 +42,7 @@ protected override void Update()
                 FadeTime = key_fade_time,
                 KeyDownTextColor = KeyDownTextColor,
                 KeyUpTextColor = KeyUpTextColor,
+                IsCounting = { BindTarget = IsCounting }
```

Comment:
I'm not a huge fan of the implementations having to hook this up themselves. Ideally it should be done automatically by the base class.

Else we're 99% going to hit this same rake with every new counter implementation.